### PR TITLE
homework

### PR DESCRIPTION
**Pull request by fargoing**
python, numpy, pandas homework

### CODE REVIEW

**Comment by iphysresearch** [comment]
Good Job!!!   @fargoing
---

**Comment by iphysresearch** [comment]
@fargoing 
你需要update branch 一下，不然 actions 工作流会认为你有不止一个修改过的作业文件。https://github.com/iphysresearch/GWData-Bootcamp/pull/137#issuecomment-1845666444

**Comment by fargoing** [comment]
老师，是需要我更新下本地的branch 吗？

**Comment by iphysresearch** [comment]
Good Job!!!   @fargoing
---
Python homework:
- Total questions: 108
- Correct answers: 106
- Score: 98.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 11
- Score: 91.00%
---

**Comment by fargoing** [comment]
老师，通过看视频，我明白了，已经更新完再重新pull request了

**Comment by iphysresearch** [comment]
Good Job!!!   @fargoing
---
Python homework:
- Total questions: 108
- Correct answers: 106
- Score: 98.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 12
- Score: 100.00%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @fargoing
---
Python homework:
- Total questions: 108
- Correct answers: 107
- Score: 99.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 12
- Score: 100.00%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @fargoing
---
Python homework:
- Total questions: 108
- Correct answers: 108
- Score: 100.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 12
- Score: 100.00%
---

**Review by iphysresearch — COMMENTED** [review]
@fargoing 
作业做得很好哈！把扩展作业也做一下吧哈？

**Comment by iphysresearch** [comment]
Good Job!!!   @fargoing
---
Python homework:
- Total questions: 108
- Correct answers: 108
- Score: 100.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 12
- Score: 100.00%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @fargoing
---
Python homework:
- Total questions: 108
- Correct answers: 108
- Score: 100.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 12
- Score: 100.00%
---

File: 2023/homework/jixiang_shao/Leetcode_tasks.txt
Comment (iphysresearch) [inline 2023/homework/jixiang_shao/Leetcode_tasks.txt:1]: 完成Python编程扩展作业哈！

**Review by iphysresearch — APPROVED** [review]
@fargoing 
完成Python编程的基础和扩展作业！

**Pull request merged**